src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -68,6 +68,9 @@ Copyright (c) .NET Foundation. All rights reserved.
   -->
 
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+    <!-- If targeting netcoreapp1.1, and RuntimeFrameworkVersion is not specified, use version 1.1.1 -->
+    <RuntimeFrameworkVersion Condition="'$(RuntimeFrameworkVersion)' == '' And '$(_TargetFrameworkVersionWithoutV)' == '1.1'">1.1.1</RuntimeFrameworkVersion>
```

Comment:
I thought we were getting away from this. It sucks to have to patch this file whenever there's a patch. Also, given all of the issues with NETStandard.Libary 1.6 -> 1.6.1, I think it would be prudent to get to a place where simply upgrading the SDK never changes your package graph.